Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 
 // MARK: - Setup
 
-private let environment: Environment = .development
+private let environment: Environment = .production
```

Comment:
Changing back to production, because we cant use spm xcode because spm has local package dependencies. 😢